xendit/models/ewallet/ewallet.py

```diff
@@ -48,6 +54,45 @@ def helper_create_linkaja_item(*, id, name, price, quantity, **kwargs):
 
         return LinkAjaItem.Query(**params)
 
+    @staticmethod
+    def helper_create_basket_item(
+        *,
+        reference_id,
+        name,
+        category,
+        currency,
+        price,
+        quantity,
+        type,
+        url=None,
+        description=None,
+        sub_category=None,
+        metadata=None,
+        **kwargs,
+    ):
+        """Construct Installments Object for Charge
+
+        Args:
+          - reference_id (str)
+          - name (str)
+          - category (str)
+          - currency (str)
+          - price (int)
+          - quantity (int)
+          - type (str)
+          - url (str)
+          - description (str)
+          - sub_category (str)
+          - metadata (dict)
```

Comment:
If the document states the data type for each parameter, can we set it on the function as well?
```suggestion
        *,
        reference_id: str,
        name: str,
        category: str,
        currency: str,
        price: int,
        quantity: int,
        type: str,
        url: str =None,
        description: str =None,
        sub_category: str =None,
        metadata:dict =None,
        **kwargs,
    ):
        """Construct Installments Object for Charge

        Args:
          - reference_id (str)
          - name (str)
          - category (str)
          - currency (str)
          - price (int)
          - quantity (int)
          - type (str)
          - url (str)
          - description (str)
          - sub_category (str)
          - metadata (dict)
```

Reply:
Ooh I didn't know it could be like this, thanks for your review :+1: